Description: workaround crashes in QtQml code related to dead-store elimination
Origin: upstream, https://code.qt.io/cgit/qt/qtdeclarative.git/commit/?id=fcc2c95421710f98
Last-Update: 2016-08-29

--- a/src/qml/qml.pro
+++ b/src/qml/qml.pro
@@ -18,7 +18,7 @@
 
 greaterThan(QT_GCC_MAJOR_VERSION, 5) {
     # Our code is bad. Temporary workaround.
-    QMAKE_CXXFLAGS += -fno-delete-null-pointer-checks
+    QMAKE_CXXFLAGS += -fno-delete-null-pointer-checks -fno-lifetime-dse
 }
 
 QMAKE_DOCS = $$PWD/doc/qtqml.qdocconf
